_pytest/skipping.py

```diff
@@ -261,7 +261,7 @@ def pytest_runtest_makereport(item, call):
             else:
                 rep.outcome = "passed"
                 rep.wasxfail = explanation
-    elif item._skipped_by_mark and rep.skipped and type(rep.longrepr) is tuple:
+    elif getattr(item, '_skipped_by_mark', False) and rep.skipped and type(rep.longrepr) is tuple:
```

Comment:
Couldn't it be` isinstance(rep.longrepr,  tuple)` ?

Reply:
It could be, but I think it is fine to only expect only `tuple` types (as opposed to also expect tuple subclasses).